shared_linux/TestStandCommHost/TestStandCommHost.cxx

```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
+#include "TestStandCommHost.h"
+
+#define GET_BYTE(_x, _n) (uint8_t)(((_x) >> ((_n)*8)) & 0x00FF)
+
+TestStandCommHost::TestStandCommHost(SerialDevice& device) : TestStandComm(device)
+{
+    // Nothing else to do
+}
+
+bool TestStandCommHost::ping()
+{
+    return this->send_basic_msg(MSG_ID_PING);
+}
+
+bool TestStandCommHost::get_status()
+{
+    return this->send_basic_msg(MSG_ID_STATUS);
+}
+
+bool TestStandCommHost::home()
+{
+    return this->send_basic_msg(MSG_ID_HOME);
+}
+
+bool TestStandCommHost::move(uint16_t accel, uint16_t hold_vel, uint16_t dist, AxisId axis, Direction dir)
+{
+    // Transmit in big endian order
+    uint8_t data[8];
+    data[0] = GET_BYTE(accel, 1);
+    data[1] = GET_BYTE(accel, 0);
+    data[2] = GET_BYTE(hold_vel, 1);
+    data[3] = GET_BYTE(hold_vel, 0);
+    data[4] = GET_BYTE(dist, 1);
+    data[5] = GET_BYTE(dist, 0);
+    data[6] = (uint8_t)axis;
```

Comment:
Replace `1` with `AXIS_Y` and `0` with `AXIS_X` for clarity.

Reply:
1 and 0 there are referring to first byte and second byte of the distance, not to the axis